app/routes/wow/upload-timers/index.tsx

```diff
@@ -0,0 +1,67 @@
+import type { LoaderFunction } from '@remix-run/cloudflare'
+import { json } from '@remix-run/cloudflare'
+import { PageWrapper } from '~/components/Common'
+import type {
+  UploadTimersResponse,
+  UploadTimersItem
+} from '~/requests/WoW/UploadTimers'
+import UploadTimersRequest from '~/requests/WoW/UploadTimers'
+import { useLoaderData } from '@remix-run/react'
+import NoResults from '~/components/Common/NoResults'
+import SmallTable from '~/components/WoWResults/FullScan/SmallTable'
+import type { ColumnList } from '~/components/types'
+
+export const loader: LoaderFunction = async ({ request }) => {
+  return await json(UploadTimersRequest())
```

Comment:
You probably don't need to use the `json` function here as your request function returns a `Promise<Response>` already!

Reply:
You're right, I've removed the json function! 👍 
I can't figure out why I can't get anything from the fetch method in the `app/requests/WoW/UploadTimers.ts` file - the function works when I send the mock JSON data used in the issue, though.

If you can point me in the correct direction that would be great, I feel like I'm getting very close to something working 😅

Reply:
Try adding back in the await keyword